print-apps/oereb/general_info_and_disclaimer.jrxml

```diff
@@ -0,0 +1,65 @@
+	<property name="com.jaspersoft.studio.data.defaultdataadapter" value="One Empty Record"/>
+	<property name="com.jaspersoft.studio.unit." value="mm"/>
+	<style name="Default" isDefault="true" fontName="Cadastra" fontSize="8"/>
+	<parameter name="GeneralInformation" class="java.lang.String"/>
+	<parameter name="GeneralInformationLabel" class="java.lang.String"/>
+	<parameter name="DisclaimerGrundbuch_Title" class="java.lang.String"/>
+	<parameter name="DisclaimerGrundbuch_Content" class="java.lang.String"/>
+	<parameter name="SUBREPORT_DIR" class="java.lang.String"/>
+	<title>
+		<band height="47">
+			<textField textAdjust="StretchHeight" isBlankWhenNull="true">
+				<reportElement positionType="Float" stretchType="ContainerBottom" x="0" y="11" width="230" height="36" isPrintWhenDetailOverflows="true" uuid="d8faf59a-0317-4cf0-9ffd-49759fee539e">
+					<property name="com.jaspersoft.studio.unit.y" value="px"/>
+				</reportElement>
+				<textElement>
+					<font fontName="Cadastra" size="6"/>
+					<paragraph lineSpacingSize="11.0"/>
+				</textElement>
+				<textFieldExpression><![CDATA[$P{GeneralInformation}]]></textFieldExpression>
+			</textField>
+			<textField isBlankWhenNull="false">
+				<reportElement x="0" y="0" width="230" height="11" isPrintWhenDetailOverflows="true" uuid="361fca0e-48b8-4d23-863d-4cdb82c500a2">
+					<property name="com.jaspersoft.studio.unit.width" value="pixel"/>
+					<property name="com.jaspersoft.studio.unit.height" value="mm"/>
+					<property name="com.jaspersoft.studio.unit.x" value="pixel"/>
+					<property name="com.jaspersoft.studio.unit.y" value="mm"/>
+				</reportElement>
+				<textElement markup="styled">
+					<font fontName="Cadastra" size="6" isBold="true"/>
+					<paragraph lineSpacingSize="8.0"/>
+				</textElement>
+				<textFieldExpression><![CDATA[$P{GeneralInformationLabel}]]></textFieldExpression>
+			</textField>
+		</band>
+	</title>
+	<pageHeader>
+		<band height="55">
+			<textField>
+				<reportElement x="0" y="6" width="230" height="11" uuid="b1952a7b-2006-4a21-bf98-8162489e3bac">
+					<property name="com.jaspersoft.studio.unit.width" value="mm"/>
+					<property name="com.jaspersoft.studio.unit.x" value="mm"/>
+					<property name="com.jaspersoft.studio.unit.y" value="mm"/>
+					<property name="com.jaspersoft.studio.unit.height" value="mm"/>
+				</reportElement>
+				<textElement>
+					<font fontName="Cadastra" size="6" isBold="true"/>
+				</textElement>
+				<textFieldExpression><![CDATA[$P{DisclaimerGrundbuch_Title}]]></textFieldExpression>
```

Comment:
and here:
`<textFieldExpression><![CDATA[$P{DisclaimerLandRegister_Title}]]></textFieldExpression>`